### ISSUE TITLE
configWinVS14.bat failes

### ISSUE THREAD

**Issue by it-cosmos** [OpenModelica/OMFMISimulator#39]
although I have installed the requierd tools (cmake,boost and FMIL) the script fails:

```
C:\Users\Lutz\github\OMFMISimulator>configWinVS14.bat
# config fmil
CMake Error: The source directory "C:/Users/Lutz/github/OMFMISimulator/3rdParty/FMIL" **does not appear to contain CMakeLists.txt.**
Specify --help for usage, or press the help button on the CMake GUI.
# build fmil
Das System kann den angegebenen Pfad nicht finden.
Der Befehl "msbuild.exe" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
# build Lua
Das System kann den angegebenen Pfad nicht finden.
# build OMFMISimulator
Das System kann den angegebenen Pfad nicht finden.
Der Befehl "msbuild.exe" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
Drücken Sie eine beliebige Taste . . .
# config OMFMISimulator
CMake Error: The source directory "C:/Users/Lutz" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
Drücken Sie eine beliebige Taste . . .
```

**Comment by lochel** [comment]
I guess you did not check out the subprojects, i.e. 3rdParty and testsuite. You can use `git clone ... --recursive` to fetch all submodules while cloning a repository or `git submodule init 3rdParty/ && git submodule update 3rdParty/` to just fetch the 3rdParty module within the already checked out OMFMISimulator repository.

**Comment by it-cosmos** [comment]
Correct, but I run into a new problem running configWinVS14.bat:
`Der Befehl ""C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat"" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
Microsoft (R)-Buildmodul, Version 14.0.25420.1
Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.
`
I have installed VS 2015 express version. Recherege in the internet I found in [https://stackoverflow.com/questions/43372235/vcvarsall-bat-for-visual-studio-2017](url).
To summarize, sth along the lines of

    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64

or

    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\vsdevcmd" -arch=x64

---
In my case it is the vsdevcmd.bat file

**Comment by it-cosmos** [comment]
sorry, the correct link is [here](https://stackoverflow.com/questions/43372235/vcvarsall-bat-for-visual-studio-2017)

**Comment by arun3688** [comment]
@it-cosmos 
The microsoft visual studio 2015 does not install the visual tool c++ by default, please follow the link for vcvarsall.bat to be installed

https://social.msdn.microsoft.com/Forums/en-US/1071be0e-2a46-4c30-9546-ea9d7c4755fa/where-is-vcvarsallbat-file?forum=visualstudiogeneral

**Comment by lochel** [comment]
I reopen this issue to keep track of it.

**Comment by it-cosmos** [comment]
I have installed 

> en_visual_studio_community_2015_with_update_3_x86_x64_web_installer_8922963.exe

 with the suggested tools as describe in the link above. Thers is still no 

> vcvarsall.bat

 file.
Also now VC2017 is the latest version, to be able to install the 2015 version I had to register in the Visual Studio community. So I suggest also to add support for VC2017.

**Comment by lochel** [comment]
We do not support VS 2017 yet, because there are issues with VS 2017 and boost.

This might be helpful if you still cannot find the vcvarsall.bat:
https://social.msdn.microsoft.com/Forums/en-US/1071be0e-2a46-4c30-9546-ea9d7c4755fa/where-is-vcvarsallbat-file?forum=visualstudiogeneral
